### DISCUSSION TITLE
state.toStrings() should be state.path?

### DISCUSSION THREAD

**Discussion by NikitaIT** [statelyai/xstate#1299] [General] [answered]
Why is the name chosen based on the data type and not the content?

https://xstate.js.org/docs/guides/states.html#state-tostrings
This method returns an array of strings that represent all of the **state value paths**. For example, assuming the current state.value is `{ red: 'stop' }`.

**Comment by davidkpiano** [comment, marked as answer]
Because "path" is not accurate. A state doesn't have a single path; it is the combination of many state nodes (the state's "configuration"), each of which has a path. So if you want to rename it, it would be `state.paths`.
